l10n/en-US/viewer.ftl

```diff
@@ -0,0 +1,386 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+## Main toolbar buttons (tooltips and alt text for images)
+
+pdfjs-previous-button =
+    .title = Previous Page
+pdfjs-previous-button-label = Previous
+pdfjs-next-button =
+    .title = Next Page
+pdfjs-next-button-label = Next
+
+# .title: Tooltip for the pageNumber input.
+pdfjs-page-input =
+    .title = Page
+
+# Variables:
+#   $pagesCount (Number) - the total number of pages in the document
+pdfjs-of-pages = of { $pagesCount }
```

Comment:
Can you help me understand where this is used? I wonder why it starts lowercase, and what's before it.

Reply:
![image](https://github.com/mozilla/pdf.js/assets/5641725/4d8307c5-be09-4cf0-b87b-19642e0cc94e)